src/Rest/SendEmailRoute.php

```diff
@@ -0,0 +1,201 @@
+{
+
+	/**
+	 * Route slug
+	 *
+	 * @var string
+	 */
+	public const ENDPOINT_SLUG = '/send-email';
+
+	public const TO_PARAM                          = 'emailTo';
+	public const SUBJECT_PARAM                     = 'emailSubject';
+	public const MESSAGE_PARAM                     = 'emailMessage';
+	public const ADDITIONAL_HEADERS_PARAM          = 'emailAdditionalHeaders';
+	public const SEND_CONFIRMATION_TO_SENDER_PARAM = 'emailSendCopyToSender';
+	public const CONFIRMATION_SUBJECT_PARAM        = 'emailConfirmationSubject';
+	public const CONFIRMATION_MESSAGE_PARAM        = 'emailConfirmationMessage';
+	public const EMAIL_PARAM                       = 'email';
+
+	/**
+	 * Basic Captcha object.
+	 *
+	 * @var BasicCaptcha
+	 */
+	protected $basicCaptcha;
+
+	/**
+	 * Construct object.
+	 *
+	 * @param BasicCaptcha $basicCaptcha Basic captcha object.
+	 */
+	public function __construct(BasicCaptcha $basicCaptcha)
+	{
+		$this->basicCaptcha = $basicCaptcha;
+	}
+
+	/**
+	 * Method that returns rest response.
+	 *
+	 * @param  \WP_REST_Request $request Data got from endpoint url.
+	 *
+	 * @return WP_REST_Response|mixed If response generated an error, WP_Error, if response
+	 *                                is already an instance, WP_HTTP_Response, otherwise
+	 *                                returns a new WP_REST_Response instance.
+	 */
+	public function routeCallback(\WP_REST_Request $request)
+	{
+
+		try {
+			$params = $this->verifyRequest($request);
+		} catch (UnverifiedRequestException $e) {
+			return rest_ensure_response($e->getData());
+		}
+
+	  // If email was sent (and sending a copy back to sender is enabled) we need to validate this email is correct.
+		if (
+			$this->shouldSendEmailCopyToUser($params) &&
+			! $this->isEmailSetAndValid($params)
+		) {
+			return $this->restResponseHandler('invalid-email-error');
+		}
+
+		$emailInfo            = $this->buildEmailInfoFromParams($params);
+		$emailInfo['headers'] = $this->addDefaultHeaders($emailInfo['headers']);
+		$response              = wp_mail($emailInfo['to'], $emailInfo['subject'], $emailInfo['message'], $emailInfo['headers']);
+
+	  // If we need to send copy to sender.
+		if ($this->shouldSendEmailCopyToUser($params)) {
+			$emailConfirmationInfo = $this->buildEmailInfoFromParams($params, true);
+			$responseConfirmation   = wp_mail($params[self::EMAIL_PARAM], $emailConfirmationInfo['subject'], $emailConfirmationInfo['message']);
+		}
+
+		if (
+			! $response ||
+			( $this->shouldSendEmailCopyToUser($params) && empty($responseConfirmation) )
+		) {
+			return $this->restResponseHandler('send-email-error');
+		}
+
+		return \rest_ensure_response([
+			'code' => 200,
+			'message' => esc_html__('Email sent', 'eightshift-forms'),
+			'data' => [],
+		]);
+	}
+
+	/**
+	 * Adds default email headers so email is interpreted as HTML.
+	 *
+	 * @param  string $headers Existing headers.
+	 * @return string
+	 */
+	protected function addDefaultHeaders(string $headers): string
```

Comment:
Is this class being extended? If not then the internal methods should be `private`.